src/lib/FlightTasks/tasks/ManualPositionSmoothVel/FlightTaskManualPositionSmoothVel.cpp

```diff
@@ -100,25 +102,7 @@ void FlightTaskManualPositionSmoothVel::_updateSetpoints()
 		_smoothing[2].setMaxVel(_constraints.speed_down);
 	}
 
-	Vector2f vel_xy_sp = Vector2f(&_velocity_setpoint(0));
 	float jerk[3] = {_jerk_max.get(), _jerk_max.get(), _jerk_max.get()};
-	float jerk_xy = _jerk_max.get();
-
-	if (_jerk_min.get() > _jerk_max.get()) {
-		_jerk_min.set(0.f);
-	}
-
-	if (_jerk_min.get() > FLT_EPSILON) {
```

Comment:
So _jerk_min is not used anymore?

Reply:
Correct. Several people reported that the logic felt weird so I decided to remove it until I find a better approach.

Reply:
Ok, would be nice if we had a good way of dealing with this.